config/prod/test-ahayden-rds-postgres.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision an EC2 instance
+template_path: remote/managed-rds-postgres-v1.j2
+stack_name: test-ahayden-rds-postgres
+sceptre_user_data:
+  # (Optional) Expose database ports to this whitelist of CIDR blocks
+   OpenIngress: ["207.109.76.122"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "iAtlas-testing"
+  # The resource owner
+  OwnerEmail: "aaron.hayden@sagebase.org"
+
+  # Set unique database user and password values in ssm with a path specific to the project
+  DBMasterUsername: !ssm /iAtlas/postgresql/dev/test-ahayden-rds-postgres-username
+  DBMasterUserPassword: !ssm /iAtlas/postgresql/dev/test-ahayden-rds-postgres-password
+  #
+  # Set SSL/TLS CSR values
+  DnsDomainName: "sagebionetworks.org"
+  DnsSubDomainName: "iatlasdb"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "snowflakevpc"
```

Comment:
@zaro0508 this is where I use the VPC name

Reply:
@ahayden @zaro0508 is it a good idea for other things to share `snowflakevpc` with the toil cluster? Considering the extensive permissions that the cluster leader has

Reply:
I think you're right... computevpc instead?